ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
Is it overridable in an environment or do i need to manipulate the global `group_vars/all`?